### PR TITLE
EDU-1249 Media library improvements

### PR DESCRIPTION
**Pull request by adrianaluchian**
Closes https://educandu.atlassian.net/browse/EDU-1249

Screen Shot 2023-03-21 at 10 53 48

### CODE REVIEW

**Comment by ahelmberger** [comment]
> Closes https://educandu.atlassian.net/browse/EDU-1249
> 
> Screen Shot 2023-03-21 at 10 53 48

Just a small little thing: when there's no file preview in the "first half" (so basically we see the search invitation) and the display switches to stacked when the screen width decreases, the two "halves" of the dropzone are not taking the same amount of screen width (the second gets all the rest). Maybe this is relatively easy to fix, but if not, not really a huge issue in my opinion.

**Comment by adrianaluchian** [comment]
> Just a small little thing: when there's no file preview in the "first half" (so basically we see the search invitation) and the display switches to stacked when the screen width decreases, the two "halves" of the dropzone are not taking the same amount of screen width (the second gets all the rest). Maybe this is relatively easy to fix, but if not, not really a huge issue in my opinion.

Thanks for noticing this, I managed to improve it for this scenario 👍 

Screen Shot 2023-03-22 at 14 39 45
👇 
Screen Shot 2023-03-22 at 14 38 47

**Comment by ahelmberger** [comment]
> > Just a small little thing: when there's no file preview in the "first half" (so basically we see the search invitation) and the display switches to stacked when the screen width decreases, the two "halves" of the dropzone are not taking the same amount of screen width (the second gets all the rest). Maybe this is relatively easy to fix, but if not, not really a huge issue in my opinion.
> 
> Thanks for noticing this, I managed to improve it for this scenario +1
> 
> Screen Shot 2023-03-22 at 14 39 45
> 
> 
> Screen Shot 2023-03-22 at 14 38 47

:smile: :point_right: :laughing:

File: src/components/resource-selector/media-library/media-library-search-screen.js [hunk 1]
```diff
@@ -45,7 +49,21 @@ function MediaLibrarySearchScreen({
   const user = useUser();
   const dropzoneRef = useRef();
   const { t } = useTranslation('mediaLibrarySearchScreen');
+  const mediaLibraryApiClient = useService(MediaLibraryApiClient);
```
[the remark this answers is not in the document]
Comment (adrianaluchian) [reply src/components/resource-selector/media-library/media-library-search-screen.js:52]: Indeed, using it now.

[the remark this answers is not in the document]
Comment (ahelmberger) [reply src/components/resource-selector/media-library/media-library-search-screen.js]: Poifect!

**Review by ahelmberger — APPROVED** [review]

**Pull request merged**